code/modules/mob/living/carbon/human/human.dm

```diff
@@ -269,7 +269,7 @@
 		if (pda.id)
 			return pda.id.rank ? pda.id.rank : if_no_job
 		else
-			return pda.ownrank
+			return pda.ownrank ? pda.own_rank : if_no_job

```

Comment:
own_rank should be ownrank.

Reply:
Bah, that snuck out of the commit. Fixed now